frontend_app_workspace/src/container/PopupCreateWorkspace.jsx

```diff
@@ -111,17 +167,108 @@ export class PopupCreateWorkspace extends React.Component {
+                />
+
+                <div className='newSpace__label'> {props.t("Space's type:")} </div>
+                <SingleChoiceList
+                  list={SPACE_TYPE_LIST}
+                  onChange={this.handleChangeSpacesType}
+                  currentValue={state.newType}
+                />
+
+                <div className='newSpace__button'>
+                  <button
+                    className='btn highlightBtn primaryColorBg primaryColorBorder primaryColorBgDarkenHover primaryColorBorderDarkenHover'
+                    disabled={!state.newName || state.newName.length === 0 || !state.newType || state.newType.length === 0}
+                    onClick={this.handleClickNextOrBack}
+                    title={props.t('Next')}
+                    type='button'
+                  >
+                    {props.t('Next')} <i className='fa fa-arrow-right newSpace__icon__right' />
+                  </button>
+                </div>
+              </>
+            )
+            : (
+              <>
+                <div className='newSpace__label'> {props.t('Parent space:')} </div>
+                <Select
+                  className='newSpace__input'
+                  isSearchable
+                  onChange={this.handleChangeParentSpace}
+                  options={state.parentOptions}
+                  defaultValue={state.parentOptions[0]}
+                />
+                {state.showWarningMessage && (
+                  <div className='newSpace__warningMessage'>
+                    <i className='fa fa-exclamation-triangle slowblink newSpace__icon__left' style={{ color: state.config.hexcolor }} />
+                    {props.t('Be careful, we do not recommend creating more than two levels of spaces because it makes the information much less accessible.')}
+                  </div>
+                )}
+
+                <div className='newSpace__label'> {props.t('Default role:')} </div>
+                <SingleChoiceList
+                  list={ROLE_LIST}
+                  onChange={this.handleChangeNewDefaultRole}
+                  currentValue={state.newDefaultRole}
+                />
+
+                <div className='newSpace__button'>
+                  <button
+                    className='btn newSpace__button__back highlightBtn primaryColorBg primaryColorBorder primaryColorBgDarkenHover primaryColorBorderDarkenHover'
+                    disabled={!state.newName || state.newName.length === 0 || !state.newType || state.newType.length === 0}
+                    onClick={this.handleClickNextOrBack}
+                    title={props.t('Back')}
+                    type='button'
+                  >
+                    <i className='fa fa-arrow-left newSpace__icon__left' /> {props.t('Back')}
+                  </button>
+
+                  <button
+                    className='btn highlightBtn primaryColorBg primaryColorBorder primaryColorBgDarkenHover primaryColorBorderDarkenHover'
```

Comment:
```suggestion
                    className='btn highlightBtn primaryColorBg primaryColorBorder primaryColorBgDarkenHover primaryColorBorderDarkenHover'
```
The className is similar to the previous one, would it be worth it to use a base constant? (I'm not sure, it's just a suggestion)

Reply:
Yes, I'm going to do this. Thank you.